modules/ztp-configuring-uefi-secure-boot.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-uefi-secure-boot_{context}"]
+= Configuring UEFI secure boot for clusters using PolicyGenTemplate CRs
+
+You can configure UEFI secure boot for vRAN clusters that are deployed using the GitOps ZTP pipeline.
+
+.Prerequisites
+
+* Create a Git repository where you manage your custom site configuration data.
+
+.Procedure
+
+. Create the following `MachineConfig` resource and save it in the `uefi-secure-boot.yaml` file:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: uefi-secure-boot
+spec:
+  config:
+    ignition:
+      version: 3.1.0
+  kernelArguments:
+    - efi=runtime
+----
+
+. In your custom `/siteconfig` directory, create a `/sno-extra-manifest` folder and add the `uefi-secure-boot.yaml` file, for example:
++
+[source,text]
+----
+siteconfig
+├── site1-sno-du.yaml
+├── site2-standard-du.yaml
+└── sno-extra-manifest
+    └── uefi-secure-boot.yaml
+----
+
+. In your `SiteConfig` CR, enter the directory name in the `extraManifestPath` field and set the value for `bootMode` to `UEFISecureBoot`, for example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "example-cluster"
+  namespace: "example-cluster"
+spec:
+  baseDomain: "ran.base-domain.lab"
+  pullSecretRef:
+    name: "pull-secret"
+  clusterImageSetNameRef: "img4.9.9-x86-64-appsub"
+  sshPublicKey: "<ssh_public_key>"
+  clusters:
+    - clusterName: "example-cluster"
+      extraManifestPath: siteconfig/sno-extra-manifest/
```

Comment:
I think this should be 
extraManifestPath: sno-extra-manifest/

Reply:
This example includes parent folder: https://issues.redhat.com/browse/CNF-2083?focusedCommentId=19498223&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19498223

Reply:
The example above is specific to dev environment, where they have each site in its own folder to be able to install one site at a time. However, in official ZTP doc, all the site yamls are directly under siteconfig dir, so the reference of sno-extra-manifest needs to align with the officially recommended structure. Here is an example for qe env where the repo is structured as recommended: http://registry.kni-qe-0.lab.eng.rdu2.redhat.com:3000/kni-qe/ztp-site-configs/src/edge-4.10/siteconfig/ocp-edge87.yaml#L15